src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -0,0 +1,199 @@
+package com.wikia.webdriver.testcases.infoboxbuilder;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.oasis.pages.InfoboxBuilderPage;
+import com.wikia.webdriver.elements.oasis.pages.TemplatePage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.PortableInfobox;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.themedesigner.SpecialThemeDesignerPageObject;
+import org.testng.annotations.Test;
+
+public class InfoboxBuilderTests extends NewTestTemplate {
```

Comment:
add this above `@Execute(onWikia = "mediawiki119")` so you don't need to set in on single test case